gaia/gdal_functions.py

```diff
@@ -96,7 +96,7 @@ def gdal_resize(raster, dimensions, projection, transform):
     return resized_ds
 
 
-def gdal_clip(raster_input, raster_output, polygon_json, nodata=-32768):
+def gdal_clip(raster_input, raster_output, polygon_json, nodata=0):
```

Comment:
0 is a safer default nodata value to use if none is provided (if the input datasource is of type UInt16, -32768 gets converted to positive 32768).